svy/src/helpers/balance.ts

```diff
@@ -21,4 +23,13 @@ export function sendSVY(accountAddress: Address, svySent: BigInt, block: ethereu
   account.lastUpdatedTimestamp = block.timestamp;
   account.save();
   return account;
+}
+
+export function updateVeSVYBalance(accountAddress: Address, block: ethereum.Block): Account {
```

Comment:
Move this function into a `helpers/veSVY.ts` and move the svy balance functions into a `helpers/svy.ts`